src/tests/unit/tests/injected/visualization/svg-drawer-v2.test.ts

```diff
@@ -996,7 +996,7 @@ describe('SVGDrawerV2Tests', () => {
             });
     }
 
-    function setupFilterFactoryDefault(dom) {
+    function setupFilterFactoryDefault(dom): void {
```

Comment:
>dom [](start = 39, length = 3)

Should the parameter also have a type? I'm surprised that the line passes the lint check without that

Reply:
There is no check as of yet to do that and personally I have not been touching those, So I would rather take that as a separate task with different specific PRs for those ones. 👍

Reply:
Do you mean "we haven't enabled such a check"? I think the check exists (as the `parameter` argument to the `typedef` rule)

Reply:
(regardless, fine with doing as a separate PR)

Reply:
Discussed offline, this should be discussed in our next engineering discussion meeting and hopefully come up with some solution for this 😄